PostFreeSurfer/PostFreesurferPipelineLongPrep.sh

```diff
@@ -0,0 +1,418 @@
+#!/bin/bash
+
+# # PostFreeSurferPipelineLongPrep.sh
+#
+# ## Copyright Notice
+#
+# Copyright (C) 2022-2024 The Human Connectome Project
+#
+# * Washington University in St. Louis
+# * University of Minnesota
+# * Oxford University
+#
+# ## Author(s)
+#
+# * Mikhail Milchenko, Department of Radiology, Washington University in St. Louis
+# * Matthew F. Glasser, Department of Anatomy and Neurobiology, Washington University in St. Louis
+#
+# ## Product
+#
+# [Human Connectome Project][HCP] (HCP) Longitudinal Pipelines
+#
+# ## License
+#
+# See the [LICENSE](https://github.com/Washington-University/HCPPipelines/blob/master/LICENSE.md) file
+
+set -eu
+
+pipedirguessed=0
+if [[ "${HCPPIPEDIR:-}" == "" ]]
+then
+    pipedirguessed=1
+    #fix this if the script is more than one level below HCPPIPEDIR
+    export HCPPIPEDIR="$(dirname -- "$0")/.."
+fi
+
+
+# Requirements for this script
+#  installed versions of: FSL
+#  environment: HCPPIPEDIR, FSLDIR
+
+
+source "$HCPPIPEDIR/global/scripts/newopts.shlib" "$@"
+source "$HCPPIPEDIR/global/scripts/debug.shlib" "$@"
+#pending resolution on T1w-only mode support by longitudinal scripts.
+#source "$HCPPIPEDIR/global/scripts/processingmodecheck.shlib" "$@" # Check processing mode requirements
+
+log_Msg "Platform Information Follows: "
+uname -a
+"$HCPPIPEDIR"/show_version
+
+opts_SetScriptDescription "takes cross-sectional PreFreesurfer output, as well as longitudinal FreeSurfer output folders.\
+Creates volumes and transforms analogous to those produced by PreFreeSurferPipeline, but in longitudinal template space"
```

Comment:
Backslash-newline gets removed entirely from `""` strings, so this didn't have any whitespace after the period.
```suggestion
opts_SetScriptDescription "takes cross-sectional PreFreesurfer output, as well as longitudinal FreeSurfer output folders.  \
Creates volumes and transforms analogous to those produced by PreFreeSurferPipeline, but in longitudinal template space"
```

Reply:
ok